src/metatensor/models/experimental/alchemical_model/tests/test_exported.py

```diff
@@ -25,6 +25,13 @@ def test_to(tmp_path, device, dtype):
 
     os.chdir(tmp_path)
 
+    if dtype == torch.float32:
+        dtype_string = "float32"
+    elif dtype == torch.float64:
+        dtype_string = "float64"
+    else:
+        raise ValueError(f"Unsupported dtype: {dtype}")
```

Comment:
Why do we need this test here?

Reply:
Unfortunately it's not a test. It's converting the dtype to a string, as required by the new `metatensor.torch.atomistic.ModelCapabilities`

Reply:
Ahh I see. Does it make sens to change our internal convention to strings as well. This may avoid converting it back and forth....

Reply:
I don't see why metatensor atomistic is not accepting a torch dtype. To me, that's the clean solution. But now we're working with this release so we have to adapt

Reply:
> I don't see why metatensor atomistic is not accepting a torch dtype.

Because we need to serialize it in the model, and serialized dtype are not good for user messages (`expected dtype 2, got dtype 6`. Additionally, there is no guarantee that the dtype numeric indexes will stay the same from one torch release to the next.